Gopkg.lock

```diff
@@ -193,6 +193,12 @@
   packages = ["."]
   revision = "fbc6b23ce49e2578f572d2e72bb72fa03c7145de"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/mattbaird/jsonpatch"
```

Comment:
We use https://github.com/evanphx/json-patch as the json patch pkg in k8s a lot.
I'm not sure if there is any subtle difference.